src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+document.addEventListener('DOMContentLoaded', () => {
+  const theadRow = document.querySelector('table thead tr');
+  const theadCells = theadRow.querySelectorAll('th');
+  const clonedTheadCell = theadCells[1].cloneNode(true);
```

Comment:
There is a potential issue here if the table has fewer than 2 columns. Accessing `theadCells[1]` will result in an error if there is only one or no columns. Consider adding a check to ensure there are enough columns before attempting to clone.